chrome_manifest_v3/model.js

```
@@ -1,784 +1,764 @@
+      );
+    }
+  },
 };
 
 Gmail2Trello.Model.prototype.submit = function () {
-    let self = this;
-    if (self.newCard === null) {
-        g2t_log("submit: data is empty");
-        return false;
-    }
+  let self = this;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'self' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)